js/shortcode-quicktag-button.js

```diff
@@ -111,6 +113,24 @@ window.wp_sc_buttons.qt = (function(window, document, $, QTags, buttons, scbutto
 
 			shortcode += ']';
 
+			/**
+			 * Shortcode closings for both quicktags and visual editor.
+			 */
+			if ( mce_selection ) {
+				shortcode += mce_selection + close_tag;
+			} else if ( ! btn.isVisual ) {
+				var canvas = document.getElementById( 'content' ),
+					text = canvas.value,
+					startPos = canvas.selectionStart,
+					endPos = canvas.selectionEnd;
+
+				window.console.log( startPos,endPos );
```

Comment:
is this log necessary? Also, if you're logging, use the logging method, `scbuttons.log`.